src/vue/backend/src/templates/GeneralSettings.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <idx-block className="general-settings">
+        <idx-form-group>
+            <idx-form-label for="website-wrapper"><strong>Name Your Global Website Wrapper</strong><br> Wrappers set the overall styling of your IDX Broker pages, some words about maintaining a consistent design between WordPress and IDX Broker.</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="website-wrapper"
+                :value="wrapperName"
+                @keyup="debounceInput($event)"
```

Comment:
It looks like the idx form input component takes care of this by emitting a "change" event on enter or blur. That removes our need to use debounce. 
```
@change="generalSettingsStateChange(...)"
```
http://bitbucket.hq.idx.org/projects/PROJ/repos/vuecomponentlibrary/browse/src/components/Atoms/Forms/FormInput.vue#33